src/common/compatible.ts

```diff
@@ -0,0 +1,31 @@
+import fs from 'node:fs'
+import path from 'node:path'
+import { tmpdir } from 'node:os'
+import { randomStr } from '@murongg/utils'
+
+export interface BrowserStreamParam {
+  fileType?: string
+  fileName?: string
+}
+
+export interface NodeStreamParam {
+  tmpPath?: string
+}
+
+export function createFileStream(content: any, param: BrowserStreamParam & NodeStreamParam) {
+  if (__BROWSER__) {
+    const { fileType, fileName } = param
+    if (fileType === undefined || fileName === undefined)
```

Comment:
Do we need `fileType !== '' || fileName !== ''` conditions?

Reply:
no harm i guess, you can add it